ctms/config.py

```diff
@@ -48,18 +51,10 @@ class Settings(BaseSettings):
     logging_level: LogLevel = LogLevel.INFO
     sentry_debug: bool = False
 
-    fastapi_env: Optional[str] = None
-    host: str = "0.0.0.0"
-    port: int = 8000
+    fastapi_env: Optional[str] = Field(default=None, alias="FASTAPI_ENV")
+    host: str = Field(default="0.0.0.0", alias="HOST")
+    port: int = Field(default=8000, alias="PORT")
```

Comment:
These aliases provide a way to specify these settings without the `ctms_` prefix present on the other settings